Doc/library/asyncio-stream.rst

```diff
@@ -206,12 +206,21 @@ StreamReader
 
    .. coroutinemethod:: read(n=-1)
 
-      Read up to *n* bytes.  If *n* is not provided, or set to ``-1``,
-      read until EOF and return all read bytes.
+      Read up to *n* bytes from the stream.
 
+      If *n* is not provided, or set to ``-1``,
```

Comment:
```suggestion
      If *n* is not provided or set to ``-1``,
```

This comma isn't necessary, and number of commas in this sentence harms readability and clear hierarchy